inc/item_devices.class.php

```diff
@@ -1109,20 +1107,42 @@ function copyToClipboard(item) {
             $canRead = (Session::haveRightsOr($attributs['right'], [READ, UPDATE]));
          }
 
+         if (!isset($attributs['datatype'])) {
+            $attributs['datatype'] = 'text';
+         }
+
+         echo "<td>";
+         if ($canRead) {
+            $rand = mt_rand();
+            switch ($attributs['datatype']) {
+               case 'dropdown':
+                  $fieldType = 'dropdown';
+                  break;
+
+               default:
+                  $fieldType = 'textfield';
+            }
+            echo "<label for='${fieldType}_$field$rand'>".$attributs['long name']."</label>";
+         } else {
+            echo $attributs['long name'];
+         }
+         echo "</td><td>";
+
```

Comment:
@trasher you asked whine I was working on this code to add html LABEL tags. here it is